crates/rpc/rpc/src/eth/gas_oracle.rs

```diff
@@ -92,6 +111,8 @@ pub struct GasPriceOracle<Provider> {
     ignore_price: Option<u128>,
     /// The latest calculated price and its block hash
     last_price: Mutex<GasPriceOracleResult>,
+    /// Cache storing the lowest effective tip values of recent blocks
+    lowest_effective_tip_cache: Mutex<EffectiveTipLruCache>,
```

Comment:
these should be combined into a single type.

let's call it `GasPriceOracleInner { last_price, lowest_effective_tip_cache}`